documentation/modules/ROOT/pages/integration-with-aap.adoc

```diff
@@ -0,0 +1,176 @@
+
+[#augmenting-openshift]
+=== Augmenting Openshift with Ansible Automation
+
+As discussed in previous sections, we leverage RHACM and Openshift to configure our clusters through custom resource definitions and policy based governance. These definitions define the desired state of the clusters, and upon deployment and beyond, Openshift works to bring the clusters to compliance with our policies. This is known as a _declarative_ model of configuration, and alongside GitOps and our ZTP workflow, allows for simplicity and flexibility when managing a large number clusters at scale.
+
+Often times in our declarative configuration worflow, there comes a need to configure important non-declarative components of a system such as bare metal servers, network infrastructure, or legacy systems. In these cases, we can leverage Ansible Automation Platform to manage these components through automated _imperative_ configuration. Utilizing hooks into Ansible, we can invoke Ansible Playbooks upon deployment of a cluster, when changes in policy compliance are detected, or at other significant moments in the lifecycle of clusters. This allows for efficient and automated management of both the declarative and non-declarative components of 5G infrastructure.
+
+For a deeper dive, see link:https://www.ansible.com/blog/openshift-and-ansible-bridging-the-automation-gap-for-5g-and-beyond-networks-part-1[OpenShift and Ansible: Bridging the Automation Gap for 5G and Beyond Networks - Part 1] on the Ansible blog.
+
+[#intro-to-aap-operator]
+=== Introduction to the AAP Operator
+
+The AAP operator provides us with the necessary components to host both Ansible collections (through the Automation Hub) to leverage Ansible's diverse distribution platform and Ansible automation objects (through the Automation Controller) natively through Openshift. The operator offers the custom resource definitions AutomationHub and AutomationController, whose functions are defined below.
+
+[#aap-components]
+== Defining AAP Components
+
+[#automation-hub]
+=== Automation Hub
+
+The OpenShift AAP operator's Automation Hub, defined by the custom resource definition AutomationHub, is a downstream version of link:https://github.com/ansible/galaxy_ng[Galaxy NG] which allows for the hosting of Ansible collections on a private distribution server. Collections are a distribution format for Ansible objects including playbooks, roles, modules, and plugins. We store all necessary Ansible collections through an Automation Hub deployed on OpenShift, allowing all dependencies to be hosted and managed through our local cluster. We can upload collections, manage collections, and interact with the Automation Hub through API or through its web UI deployed at automation-hub-aap.apps.hub.5g-deployment.lab.
```

Comment:
Convert automation-hub-aap.apps.hub.5g-deployment.lab to link:

https://automation-hub-aap.apps.hub.5g-deployment.lab(https://automation-hub-aap.apps.hub.5g-deployment.lab)